test/unit/newton/newton.cpp

```diff
@@ -36,7 +35,7 @@ SCENARIO("Non-linear system to solve with Newton Numerical Diff Solver", "[numer
             CAPTURE(iter_newton);
             CAPTURE(X);
             REQUIRE(iter_newton > 0);
-            REQUIRE(X[0] == Approx(1.0));
+            REQUIRE(X[0] == Catch::Approx(1.0));
```

Comment:
Note that they recommend not using this: https://github.com/catchorg/Catch2/blob/06c0e1cfab2a2fd6d01527b92422df2a8c49a8d7/docs/comparing-floating-point-numbers.md#approx

Reply:
I changed this.